site/content/docs/5.2/components/carousel.md

```diff
@@ -314,7 +314,7 @@ const carousel = new bootstrap.Carousel('#myCarousel')
 {{< bs-table >}}
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
-| `interval` | number | `5000` | The amount of time to delay between automatically cycling an item. If `false`, carousel will not automatically cycle. |
```

Comment:
Shouldn't this read:

```markdown
| `interval` | number | `5000` | The amount of time to delay between automatically cycling an item. If `0`, carousel will not automatically cycle. |
```

Reply:
Apparently if `0` the interval is just very short (no interval) so the animation is very quick. There's a cycle when `data-bs-ride` is set to `true` or `carousel`.

Reply:
I agree with Julien here. Are you ok with it @XhmikosR ? can I proceed?